packages/base/Breadcrumbs/index.js

```diff
@@ -1,34 +1,36 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import styled from 'styled-components';
 import Box from '../Box';
 import Button from '../Button';
 
-const Crumb = styled(Button)`
-  margin-right: ${p => p.theme.space[4]};
-  color: ${p => p.theme.colors.text.subtle};
-  font-size: ${p => p.theme.fontSizes[2]};
-  letter-spacing: ${p => p.theme.letterSpacings[0]};
-  text-transform: uppercase;
-  text-decoration: none;
-
-  &::before {
-    content: ${p => `'${p.separator}'`};
-    margin-right: ${p => p.theme.space[4]};
-    color: ${p => p.theme.colors.text.primary};
-  }
-
-  &:last-of-type {
-    color: ${p => p.theme.colors.accent};
-  }
-`;
-
-const Breadcrumbs = ({ crumbs, separator, ...rest }) => (
+const Breadcrumbs = ({ crumbs, crumbSx, separator, ...rest }) => (
   <Box display="flex" py="4" {...rest}>
     {crumbs.map((crumb, i) => (
-      <Crumb key={i} link={crumb.path} separator={separator} simple>
+      <Button
+        key={i}
+        link={crumb.path}
+        separator={separator}
+        simple
+        sx={{
+          '&::before': {
+            color: p => p.colors.text.primary,
+            content: `'${separator}'`,
+            mr: p => p.space[4],
+          },
+          '&:last-of-type': {
+            color: p => p.colors.accent,
+          },
+          color: p => p.colors.text.subtle,
```

Comment:
We should be able to pass these theme values directly I think?

```suggestion
          color: 'text.subtle',
```

Reply:
let me check